src/promptflow/tests/test_configs/flows/script_with_special_character/script_with_special_character.py

```diff
@@ -0,0 +1,6 @@
+from promptflow import tool
+
+@tool
+def print_special_character(input1: str) -> str:
+    # Add special character to test if file read is working.
+    return "https://www.bing.com//"
```

Comment:
Can we use `\ue000\ue001` here too to align with the assertion code?
```python
        assert "https://www.bing.com/\ue000\ue001/" in result
```

Reply:
When we use `\ue000\ue001`, the result will be `\\ue000\\ue001`

Reply:
I see.